processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -186,7 +186,8 @@ public AlphaNumericTopNMetricSpec(
       @JsonProperty("previousStop") String previousStop
   )
   {
-    super(previousStop);
+    // Set empty string as previousStop of super class otherwise we will get NPE
+    super((previousStop == null) ? "" : previousStop);
     this.previousStop = (previousStop == null) ? "" : previousStop;
```

Comment:
It seems semantics of "prevStop" null is that cliend is saying return me everything. In case of Inverse topNmetric setting it to empty string breaks that. So, we should really do the null handling in compare(..) method like it is done in LexicographicTopNMetricSpec